app/screens/map-screen/map-screen.tsx

```diff
@@ -91,8 +91,8 @@ export const MapScreen: ScreenType = ({ navigation }: Props) => {
     useCallback(() => {
       requestLocationPermission()
 
-      if (!grantedPermission) {
-        return
+      if (grantedPermission) {
```

Comment:
I think `!grantedPermission` is correct.

Reply:
Not sure how that happened 🤔  Thanks for spotting!